### PR TITLE
Update inject_bootstrap

### PR DESCRIPTION
**Pull request by vrutkovs**
Operate on files in IBIP_DIR

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -107,6 +107,10 @@ ifeq ($(ENABLE_KUBE_API),true)
 	SERVICE_REPLICAS_COUNT=1
 	AUTH_TYPE=local
 endif
+ifdef BIP_BUTANE_CONFIG
+	BOOTSTRAP_INJECT_DIR=$(ROOT_DIR)/sno-bootstrap-manifests/
```
Comment (osherdp) [inline Makefile:111]: is it "git ignored"?
if not, can we put it under a directory that is git ignored?

File: Makefile [hunk 2]
```diff
@@ -107,6 +107,10 @@ ifeq ($(ENABLE_KUBE_API),true)
 	SERVICE_REPLICAS_COUNT=1
 	AUTH_TYPE=local
 endif
+ifdef BIP_BUTANE_CONFIG
```
Comment (osherdp) [inline Makefile:110]: Something I just realized, it's missing in the documentation for env-vars
Can you add it in an appropriate place? (I think in https://github.com/openshift/assisted-test-infra/blob/009de89225024572fe99174beded1ff058fb0540/README.md#L127)

**Review by osherdp — CHANGES_REQUESTED** [review]

File: Makefile (hunk 2 above)
Comment (vrutkovs) [reply Makefile:110]: Good idea, updated

File: Makefile (hunk 1 above)
Comment (vrutkovs) [reply Makefile:111]: Fixed

**Review by osherdp — APPROVED** [review]

**Comment by osherdp** [comment]
/override ci/prow/e2e-metal-single-node-with-worker-live-iso
still broken

**Pull request merged**